src/browser_actions/popup.js

```diff
@@ -79,6 +87,12 @@ const onResetClick = () => {
     renderClippings();
 }
 
+const onDeleteClick = (id) => {
+    chrome.runtime.sendMessage({ deleteOne: true, id}, (response) => {
```

Comment:
check first comment about refactor to refactor this too :)

Also can you rename `id` to `timestamp`

Reply:
Sure thing, makes sense.